src/components/inline-inputs/inline-inputs.component.tsx

```diff
@@ -58,21 +63,28 @@ const columnWrapper = (
 };
 
 const InlineInputs = ({
+  adaptiveLabelBreakpoint,
   label,
   htmlFor,
   children = null,
   className = "",
   gutter = "none",
   inputWidth,
+  displayAsFlex = true,
```

Comment:
I think we should call this `labelInline` to match the naming convention we've used in other inputs

Reply:
Yeah I can do that if you think that's better 👍🏻 
I went for this as it's private only to this component and figured it described what it used for in the style file but going with what it is called in other inputs is a good shout 😄